test/index.test.ts

```diff
@@ -91,4 +93,11 @@ describe('module exports', () => {
     const azrael: AzraelV0FunctionInterface = renft;
     expect(!!azrael).to.be.true;
   });
+  it('deployment addresses are checksummed', () => {
+    expect(
+      RENFT_CONTRACT_DEPLOYMENTS.map(deployment =>
+        isAddress(deployment.contractAddress)
```

Comment:
Looks like this was missed.

We should be using a common operation to avoid manual errors like this. Over time, you can probably imagine our validation logic to grow in complexity (i.e. not just look at the checksum, maybe validate the `startBlock`, for example), so we should be using a function for this.

Let's remove the duplicate code and replace with an `isValidDeployment` function?

Reply:
created an `isValidDeployment` function that utilizes zod to verify deployments.